File: fix_quotes.mjs
import { readFileSync, writeFileSync } from 'fs';

const file = 'server/index.mjs';
let content = readFileSync(file, 'utf8');

// Replace all unicode smart/curly quotes with ASCII straight quotes
content = content.replace(/\u2018/g, "'");  // Left single curly quote
content = content.replace(/\u2019/g, "'");  // Right single curly quote
content = content.replace(/\u201C/g, '"');  // Left double curly quote
content = content.replace(/\u201D/g, '"');  // Right double curly quote

writeFileSync(file, content, 'utf8');
console.log('Smart quotes fixed successfully');

File: fix_quotes.py
#!/usr/bin/env python3
import re

with open('server/index.mjs', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace all curly/smart quotes with straight ASCII quotes
content = content.replace('\u2019', "'")  # Right single quote -> straight quote
content = content.replace('\u2018', "'")  # Left single quote -> straight quote  
content = content.replace('\u201d', '"')  # Right double quote -> straight quote
content = content.replace('\u201c', '"')  # Left double quote -> straight quote

with open('server/index.mjs', 'w', encoding='utf-8') as f:
    f.write(content)

print('Successfully converted all smart quotes to ASCII quotes')
